src/App.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
```

Comment:
This will need to go if you accept the below suggestion
```suggestion
import { BrowserRouter, Switch, Route } from 'react-router-dom';
```